README.rst

```diff
@@ -162,9 +162,12 @@ Compatibility
 
 This project has been tested for compatibility with:
 
-* Python 2.7, 3.5, 3.6
-* Django 1.8-1.11, 2.0
-* Wagtail 1.6-1.13, 2.0
+* Python 3.6, 3.8
+* Django 1.11, 2.0, 2.2
+* Wagtail 1.13, 2.3, 2.8
+
+It should be compatible at all intermediate versions, as well.
+If you find that it is not, please [file an issue](issues/new).
```

Comment:
This README is [reStructuredText](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html), not Markdown, so this syntax needs to be a little different:
```suggestion
If you find that it is not, please `file an issue <https://github.com/cfpb/wagtail-sharing/issues/new>`_
```